compiler/z/codegen/FPTreeEvaluator.cpp

```diff
@@ -1125,14 +1127,13 @@ OMR::Z::TreeEvaluator::floatRemHelper(TR::Node * node, TR::CodeGenerator * cg)
    outlinedSlowPath->swapInstructionListsWithCompilation();
    
    generateS390LabelInstruction(cg, TR::InstOpCode::LABEL, node, labelNotExact);
+   callNode->setChild(0, node->getFirstChild());
+   callNode->setChild(1, node->getSecondChild());
```

Comment:
You can fold these two lines into `createWithSymRef` above via the following API:
https://github.com/eclipse/omr/blob/3271be89f2df16683616fe0e38c85693cea982bb/compiler/il/OMRNode.hpp#L187

Reply:
This API increases the ref count of the child as well. The use of older API makes sure that we do not mess up with the nodes from original trees as newly created node is used to just generate a call using nativeDispatch.